.github/workflows/ci_cron_weekly.yml

```diff
@@ -122,7 +123,7 @@ jobs:
         id: build
         with:
           arch: ${{ matrix.arch }}
-          distro: ubuntu_latest
+          distro: ubuntu_rolling
```

Comment:
Is rolling too unstable? Will we see breakage that we cannot do anything about?

Reply:
Its the only one available by our workflow which has `numpy 1.22+` pre-built.

Reply:
So theoretically, we can bump it down to latest again when the distro people catch up?

Reply:
Not like we have a choice here... but we might have to rethink if this proves to be too unstable. Unless @olebole has a better idea?

Reply:
We can use the `base_image` option and pick a docker image from docker hub that works. I think the debian bookworm image should work.

Reply:
Note `rolling` is not the unstable `devel` release. `rolling` is the latest release of the Ubuntu `23.04` lunar.